repositories.yaml

```
@@ -1,502 +1,460 @@
+    ping_files:
+      - components/support/sync-telemetry/pings.yaml
+    library_names:
+      - org.mozilla.components:support-sync-telemetry
+  - v1_name: engine-gecko
+    description: GeckoView metrics
+    notification_emails:
+      - frank@mozilla.com
+      - mdroettboom@mozilla.com
+      - android-components-team@mozilla.com
+      - geckoview-team@mozilla.com
+    url: https://github.com/mozilla/gecko-dev
+    metrics_files:
+      - toolkit/components/telemetry/geckoview/streaming/metrics.yaml
+    branch: release
+    library_names:
+      - org.mozilla.components:browser-engine-gecko
+  - v1_name: engine-gecko-beta
+    description: GeckoView metrics
+    notification_emails:
+      - frank@mozilla.com
+      - mdroettboom@mozilla.com
+      - android-components-team@mozilla.com
+      - geckoview-team@mozilla.com
+    url: https://github.com/mozilla/gecko-dev
+    metrics_files:
+      - toolkit/components/telemetry/geckoview/streaming/metrics.yaml
+    branch: beta
+    library_names:
+      - org.mozilla.components:browser-engine-gecko-beta
+  - v1_name: engine-gecko-nightly
+    description: GeckoView metrics
+    notification_emails:
+      - frank@mozilla.com
+      - mdroettboom@mozilla.com
+      - android-components-team@mozilla.com
+      - geckoview-team@mozilla.com
+    url: https://github.com/mozilla/gecko-dev
+    metrics_files:
+      - toolkit/components/telemetry/geckoview/streaming/metrics.yaml
+    library_names:
+      - org.mozilla.components:browser-engine-gecko-nightly
+  - v1_name: logins-store
+    description: >
+      A collection of Android libraries to build browsers or browser-like
+      applications
+    notification_emails:
+      - rfkelly@mozilla.com
+      - lina@mozilla.com
+      - sync-team@mozilla.com
+    url: https://github.com/mozilla/application-services
+    metrics_files:
+      - components/logins/android/metrics.yaml
+    branch: main
+    library_names:
+      - org.mozilla.appservices:logins
+  - v1_name: support-migration
+    description: >
+      Helper code to migrate from a Fennec-based (Firefox for Android) app to
+      an Android Components based app
+    notification_emails:
+      - frank@mozilla.com
+    url: https://github.com/mozilla-mobile/android-components
+    metrics_files:
+      - components/support/migration/metrics.yaml
+    ping_files:
+      - components/support/migration/pings.yaml
+    library_names:
+      - org.mozilla.components:support-migration
+  - v1_name: android-places
+    description: >
+      A collection of Android libraries to build browsers or browser-like
+      applications
+    notification_emails:
+      - frank@mozilla.com
+      - sync-team@mozilla.com
+    url: https://github.com/mozilla/application-services
+    metrics_files:
+      - components/places/android/metrics.yaml
+    branch: main
+    library_names:
+      - org.mozilla.components:places
+application_families:
+  - app_name: firefox_desktop
```

Wondering the same thing here.

```suggestion
  - name: firefox_desktop
```